docs/platforms/android/releases/index.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: Release Channels
```

Comment:
We've adapted the naming of how Google names [their Dart/Flutter releases](https://github.com/flutter/flutter/blob/master/docs/releases/Flutter-build-release-channels.md)

I'm not opposed to changing it to but it's already an established naming (at least in one part of the mobile world) so something to consider as well

Reply:
I think "Statuses" does not convey the actual meaning of what is being described

Reply:
@lizokm thanks for the call-out! I agree that we need to make sure it's not mistaken for the Releases/Release Health product. As @buenaflor and @brustolin point out, Release Channels is a somewhat established term for what we're describing here which is why I chose it. Wdyt about changing it to `SDK Release Channels`?

Reply:
@philipphofmann nope, only specific Mobile SDKs. that said, ppl are viewing this page as part of these Mobile SDK platform's docs, so feels like that would be a bit verbose, and it might be too long for the menu. but no strong opinion against it

Reply:
imo at least a search engine result should reflect that it's mobile only

e.g when a user searches for sentry sdk release channels it should show `Mobile SDK Release Channels` instead of only `SDK Release Channels` 

but not sure where this change would need to be added to affect those results

Reply:
renamed to `Mobile SDK Releases` - wdyt?